### ISSUE TITLE
progressbar prints ETA randomly when checking if it's pickable

### ISSUE THREAD

**Issue by brando90** [uqfoundation/dill#445]
When I check pickable objects I get that it prints the progressbar:
```
```
widgets = [
        progressbar.Percentage(),
        ' ', progressbar.SimpleProgress(format=f'({progressbar.SimpleProgress.DEFAULT_FORMAT})'),
        ' ', progressbar.Bar(),
        ' ', progressbar.Timer(), ' |',
        ' ', progressbar.ETA(), ' |',
        ' ', progressbar.AdaptiveTransferSpeed(unit='it'),
    ]
bar = progressbar.ProgressBar(widgets=widgets, max_value=progressbar.UnknownLength)
not dill.pickles(bar)
```
see
```
N/A% (0 of N/A) |#        | Elapsed Time: 0:00:00 | ETA:  --:--:-- |   0.0 s/it
```

how do I stop it?

cross ref: https://github.com/WoLpH/python-progressbar/issues/263


related:
- https://github.com/WoLpH/python-progressbar/issues/263
- https://github.com/uqfoundation/dill/issues/445
- https://stackoverflow.com/questions/70762899/how-does-one-stop-progressbar-from-printing-eta-progress-when-checking-if-the

**Comment by mmckerns** [comment]
It would help if you update your code snippet to be self-contained, so someone can more easily test it.  However note, as posted, this seems like a question for the `python-progressbar` developers... so I'm closing this, and we should just use the issue that has been opened there.

**Comment by mmckerns** [comment]
No imports?

**Comment by wolph** [comment]
This issue has been fixed in progressbar version 4.2.0 :)
